python/cleanup/main.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Sep 13 08:56:23 2017
+
+@author: Tatiana Kim
+"""
+
+import pandas as pd
+import numpy as np
+import utilities as ut
+import transform_records as tr
+import time as tm
+
+
+# import data 
+rawData = pd.read_csv('C:/Users/Tatiana Kim/Dropbox (Iridium Insights)/Local Folder Tatiana/ML/ParkingProject/BANES_Historic_Car_Park_Occupancy.csv') 
+
+# view the first 10 rows of the data
+rawData.head(10)
+
+# Just wanted to time this operation for reference (use time package in python 
+# for this)
+t = tm.time()
+# transform the data so that it is in a format suitable for further analysis
+OccupancyDF = tr.refine(rawData, 1.1)
+elapsed = tm.time() - t
+
+# Just want to extract Date and Time from LastUpdate - 
+# we will do some joins on Date, so need this as a separate column
+OccupancyDF['Date'] = OccupancyDF['LastUpdate'].dt.date
+OccupancyDF['Time'] = OccupancyDF['LastUpdate'].dt.time
+
+
+##########################################################################
+#  Read external data:
+
+RugbyDF = pd.read_csv('Rugby_events.csv') 
```

Comment:
Can we pretend that these csv files exist in a folder called `../../data/`. see above.